src/modules/auto-native-share.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @author Deux Huit Huit
+ *
+ * Auto native share form
+ * Uses https://github.com/WICG/web-share
+ */
+(function ($, global, undefined) {
+
+	'use strict';
+
+	var site = $('#site');
+	var sels = {
+		item: '.js-native-share'
+	};
+	var SUPPORTED = !!window.navigator.share;
+	var scope = $();
+
+	var onClick = function (e) {
+		window.navigator.share({
+			title: document.title,
+			url: document.location.href || ('' + document.location)
```

Comment:
We should add the text property too ! https://wicg.github.io/web-share/#sharedata-dictionary

Maybe we pass the value on a data-attr on the clicked button ? (it's should be optional tho since the spec says so.)

Reply:
Yeah, image also. But I would merge this without and add another PR when the code is implemented ;)

Reply:
Alright!